cms/djangoapps/contentstore/views/item.py

```diff
@@ -169,6 +169,7 @@ def xblock_handler(request, usage_key_string):
                 prereq_usage_key=request.json.get('prereqUsageKey'),
                 prereq_min_score=request.json.get('prereqMinScore'),
                 publish=request.json.get('publish'),
+                fields=request.json.get('fields'),
```

Comment:
Add documentation for this new field above.

I also if there is a more descriptive name than "fields". Perhaps the documentation will clarify that for me. :)